src/collector/base.rs

```diff
@@ -215,10 +215,6 @@ impl<'a> Run<'a> {
                 if let Some(repository) = rrdp.load_repository(rrdp_uri)? {
                     return Ok(Some(Repository::rrdp(repository)))
                 }
-                warn!(
```

Comment:
The comment above this function needs an update, as with the removal of this `warn!` there is no diagnostic information logged at all anymore (in this place).